Support harfbuzz-sys 0.5.0

With 0.5.0, some functions have been renamed and Debug is no longer derived for a few structs, resulting in compile failures.
Pinning to 0.4.1 works.

```rust
error[E0425]: cannot find function `hb_ot_font_set_funcs` in crate `hb`
    --> /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz_rs-1.1.2/src/font.rs:121:17
     |
121  |               hb::hb_ot_font_set_funcs(raw_font);
     |                   ^^^^^^^^^^^^^^^^^^^^ help: a function with a similar name exists: `hb_font_set_funcs`
     | 
    ::: /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz-sys-0.5.0/src/bindings.rs:2066:5
     |
2066 | /     pub fn hb_font_set_funcs(
2067 | |         font: *mut hb_font_t,
2068 | |         klass: *mut hb_font_funcs_t,
2069 | |         font_data: *mut ::std::os::raw::c_void,
2070 | |         destroy: hb_destroy_func_t,
2071 | |     );
     | |______- similarly named function `hb_font_set_funcs` defined here

error[E0277]: `harfbuzz_sys::_hb_var_int_t` doesn't implement `std::fmt::Debug`
  --> /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz_rs-1.1.2/src/buffer.rs:60:5
   |
60 |     var: hb::hb_var_int_t,
   |     ^^^^^^^^^^^^^^^^^^^^^ `harfbuzz_sys::_hb_var_int_t` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
   |
   = help: the trait `std::fmt::Debug` is not implemented for `harfbuzz_sys::_hb_var_int_t`
   = note: required because of the requirements on the impl of `std::fmt::Debug` for `&harfbuzz_sys::_hb_var_int_t`
   = note: required for the cast to the object type `dyn std::fmt::Debug`
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `harfbuzz_sys::_hb_var_int_t` doesn't implement `std::fmt::Debug`
   --> /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz_rs-1.1.2/src/buffer.rs:107:5
    |
107 |     var1: hb::hb_var_int_t,
    |     ^^^^^^^^^^^^^^^^^^^^^^ `harfbuzz_sys::_hb_var_int_t` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
    |
    = help: the trait `std::fmt::Debug` is not implemented for `harfbuzz_sys::_hb_var_int_t`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&harfbuzz_sys::_hb_var_int_t`
    = note: required for the cast to the object type `dyn std::fmt::Debug`
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: `harfbuzz_sys::_hb_var_int_t` doesn't implement `std::fmt::Debug`
   --> /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz_rs-1.1.2/src/buffer.rs:108:5
    |
108 |     var2: hb::hb_var_int_t,
    |     ^^^^^^^^^^^^^^^^^^^^^^ `harfbuzz_sys::_hb_var_int_t` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
    |
    = help: the trait `std::fmt::Debug` is not implemented for `harfbuzz_sys::_hb_var_int_t`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&harfbuzz_sys::_hb_var_int_t`
    = note: required for the cast to the object type `dyn std::fmt::Debug`
    = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,6 +25,6 @@ build-native-harfbuzz = ["harfbuzz-sys/build-native-harfbuzz"]
 build-native-freetype = ["harfbuzz-sys/build-native-freetype"]
 
 [dependencies]
-harfbuzz-sys = { version = ">= 0.3", default-features = false }
+harfbuzz-sys = { version = ">= 0.3, <0.5", default-features = false }
 rusttype = { version = ">= 0.7, <0.9", optional = true }
 bitflags = "^1"

